### DISCUSSION TITLE
Support (2024.2) EAP of IntelliJ

### DISCUSSION THREAD

**Discussion by ice1000** [community/community#124381] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hi, when shall I expect the update for the 2024.2 EAP IntelliJ?

If the development dependency is an issue, setting the IDE version to be empty will make the plugin build with latest snapshot, which is a foetus of the next release

**Comment by Scott-G** [comment]
2 weeks and still no response to this!?

This is totally unacceptable for a subscription service. If this continues, I will cancelling my subscription.😡

**Comment by Reddishye** [comment]
Im really frustrated with this, Im actually with the student benefits, but my plan was paying the subscription when I finished my studies, however, this continuing like this feels like Mojang developers taking a nap instead of coding, what makes me really think I should not pay the subscription when I end studying and also not recommend this to my friends. Maybe is better buying IntelliJ AI Assistant?

**Comment by ice1000** [comment, marked as answer]
Thank you! [New version](https://plugins.jetbrains.com/plugin/17718-github-copilot/versions/stable/555539) has been published
